handlers/main.yml

```
@@ -3,11 +3,14 @@
 - name: start nginx
   service: name=nginx state=started enabled=yes use=service
   become: true
+  when: nginx_install_only == false
```

Can we use `when: not`
```suggestion
  when:  not nginx_install_only
```